lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -50,7 +62,7 @@
   %endif
 
     <div class="request-response msg msg-confirm copy" id="report-request-response"></div>
-    <div class="request-response-error msg msg-warning copy" id="report-request-response-error"></div>
+    <div class="request-response-error msg msg-error copy" id="report-request-response-error"></div>
```

Comment:
@sarina With `msg-error` styling the error message looks like this:

![error-error-message](https://cloud.githubusercontent.com/assets/961441/9312091/dd21e214-451a-11e5-8d92-3a99f46cffd1.png)

I can't find any code in `data_download.coffee` that would need the `msg-warning` class to be present on the div. The code consistently uses `request-response-error` and `msg-error` to target it.